diff --git a/apps/openmw/CMakeLists.txt b/apps/openmw/CMakeLists.txt
index 17c4e96b16903e518958c63923e4a7cc9a7d816f..bc27b9d09f5d810f2ffb6172008959b30fde6fd1 100644
--- a/apps/openmw/CMakeLists.txt
+++ b/apps/openmw/CMakeLists.txt
@@ -60,7 +60,7 @@
 
 add_openmw_dir (mwlua
     luamanagerimp object objectlists userdataserializer luaevents engineevents objectvariant loadscripts contentbindings
-    context menuscripts globalscripts localscripts playerscripts luabindings objectbindings cellbindings coremwscriptbindings
+    context menuscripts globalscripts localscripts playerscripts luabindings almsivibindings objectbindings cellbindings coremwscriptbindings
     mwscriptbindings camerabindings vfsbindings uibindings soundbindings inputbindings nearbybindings dialoguebindings
     postprocessingbindings stats recordstore debugbindings corebindings worldbindings worker landbindings magicbindings factionbindings
     classbindings itemdata inputprocessor animationbindings birthsignbindings racebindings markupbindings weatherbindings regionbindings
