diff --git a/apps/openmw/mwlua/luabindings.cpp b/apps/openmw/mwlua/luabindings.cpp
index dc20205cbe..2340a3f784 100644
--- a/apps/openmw/mwlua/luabindings.cpp
+++ b/apps/openmw/mwlua/luabindings.cpp
@@ -9,2 +9,3 @@
 
+#include "lorkhanbindings.hpp"
 #include "animationbindings.hpp"
@@ -52,2 +53,3 @@ namespace MWLua
             { "openmw.world", initWorldPackage(context) },
+            { "openmw.lorkhan", initLorkhanPackage(context) },
         };
@@ -65,2 +67,3 @@ namespace MWLua
             { "openmw.nearby", initNearbyPackage(context) },
+            { "openmw.lorkhan", initLorkhanCustomPackageLoader(context) },
         };
@@ -77,2 +80,3 @@ namespace MWLua
             { "openmw.ui", initUserInterfacePackage(context) },
+            { "openmw.lorkhan", initLorkhanPackage(context) },
         };
