diff --git a/apps/openmw/mwlua/luabindings.cpp b/apps/openmw/mwlua/luabindings.cpp
index dc20205cbe15f17230b87cea60a6c4276972f0a6..2340a3f7845c1939d132a976345792973c473c90 100644
--- a/apps/openmw/mwlua/luabindings.cpp
+++ b/apps/openmw/mwlua/luabindings.cpp
@@ -7,6 +7,7 @@
 #include "../mwbase/world.hpp"
 #include "../mwworld/datetimemanager.hpp"
 
+#include "lorkhanbindings.hpp"
 #include "animationbindings.hpp"
 #include "camerabindings.hpp"
 #include "cellbindings.hpp"
@@ -50,6 +51,7 @@
             { "openmw.core", initCorePackage(context) },
             { "openmw.types", initTypesPackage(context) },
             { "openmw.world", initWorldPackage(context) },
+            { "openmw.lorkhan", initLorkhanPackage(context) },
         };
     }
 
@@ -63,6 +65,7 @@
             { "openmw.core", initCorePackage(context) },
             { "openmw.types", initTypesPackage(context) },
             { "openmw.nearby", initNearbyPackage(context) },
+            { "openmw.lorkhan", initLorkhanCustomPackageLoader(context) },
         };
     }
 
@@ -75,6 +78,7 @@
             { "openmw.input", initInputPackage(context) },
             { "openmw.postprocessing", initPostprocessingPackage(context) },
             { "openmw.ui", initUserInterfacePackage(context) },
+            { "openmw.lorkhan", initLorkhanPackage(context) },
         };
     }
