diff --git a/components/lua/scriptscontainer.cpp b/components/lua/scriptscontainer.cpp
index 8b62eb7bd4..708f5fc262 100644
--- a/components/lua/scriptscontainer.cpp
+++ b/components/lua/scriptscontainer.cpp
@@ -56,4 +56,4 @@ namespace LuaUtil
     {
-        if (!package.is<sol::userdata>())
-            throw std::logic_error("Expected package to be read-only: " + packageName);
+        if (!package.is<sol::userdata>() && !package.is<sol::function>())
+            throw std::logic_error("Expected package to be read-only or a per-script loader: " + packageName);
         mAPI.insert_or_assign(std::move(packageName), std::move(package));
