diff --git a/apps/openmw/mwsound/ffmpegdecoder.hpp b/apps/openmw/mwsound/ffmpegdecoder.hpp
index 17f1301c3ed18bcc83b203850ca08ceec1244c17..b7bfcc476992a9fcbe2f9f50e2d03c5a13e7bf90 100644
--- a/apps/openmw/mwsound/ffmpegdecoder.hpp
+++ b/apps/openmw/mwsound/ffmpegdecoder.hpp
@@ -30,6 +30,7 @@
 #include <components/files/istreamptr.hpp>
 
 #include <string>
+#include <string_view>
 
 #include "sounddecoder.hpp"
 
@@ -105,6 +106,7 @@
         size_t readAVAudioData(void* data, size_t length);
 
         void open(VFS::Path::NormalizedView fname) override;
+        void open(Files::IStreamPtr dataStream, std::string_view name);
         void close() override;
 
         std::string getName() override;
