diff --git a/apps/openmw/mwbase/luamanager.hpp b/apps/openmw/mwbase/luamanager.hpp
index bddfc3079a40c372d3725e206591fc5fa95a0ba7..a207873e006aa79fb586a9b74977406e6fd093ce 100644
--- a/apps/openmw/mwbase/luamanager.hpp
+++ b/apps/openmw/mwbase/luamanager.hpp
@@ -91,9 +91,16 @@
             = 0;
         virtual void exteriorCreated(MWWorld::CellStore& cell) = 0;
         virtual void actorDied(const MWWorld::Ptr& actor) = 0;
+        virtual void onLorkhanLockpick(const MWWorld::Ptr& actor) = 0;
+        virtual void onLorkhanActorResurrected(const MWWorld::Ptr& actor) = 0;
+        virtual void onLorkhanDialogueChoice(const MWWorld::Ptr& actor, const std::string& text) = 0;
         virtual void onDialogueResponse(
             const MWWorld::Ptr& actor, const ESM::DialInfo& info, const ESM::Dialogue& record)
             = 0;
+        virtual void onLorkhanItemPickup(const MWWorld::Ptr& player, const MWWorld::Ptr& item,
+            int count, const MWWorld::Ptr& source) = 0;
+        virtual void onLorkhanSpellCast(const MWWorld::Ptr& caster, const MWWorld::Ptr& target,
+            const std::string& spellId, const std::string& spellName) = 0;
         virtual void questUpdated(const ESM::RefId& questId, int stage) = 0;
         // `arg` is either forwarded from MWGui::pushGuiMode or empty
         virtual void uiModeChanged(const MWWorld::Ptr& arg) = 0;
