diff --git a/apps/openmw/mwworld/actiontake.cpp b/apps/openmw/mwworld/actiontake.cpp
index 1f37499e8f..04e673a1b7 100644
--- a/apps/openmw/mwworld/actiontake.cpp
+++ b/apps/openmw/mwworld/actiontake.cpp
@@ -3,2 +3,3 @@
 #include "../mwbase/environment.hpp"
+#include "../mwbase/luamanager.hpp"
 #include "../mwbase/mechanicsmanager.hpp"
@@ -40,2 +41,3 @@ namespace MWWorld
         setTarget(newitem);
+        MWBase::Environment::get().getLuaManager()->onLorkhanItemPickup(actor, newitem, count, MWWorld::Ptr());
     }
