diff --git a/apps/openmw/mwworld/actionharvest.cpp b/apps/openmw/mwworld/actionharvest.cpp
index 1d9e009afe..15137f3a88 100644
--- a/apps/openmw/mwworld/actionharvest.cpp
+++ b/apps/openmw/mwworld/actionharvest.cpp
@@ -9,2 +9,3 @@
 #include "../mwbase/environment.hpp"
+#include "../mwbase/luamanager.hpp"
 #include "../mwbase/mechanicsmanager.hpp"
@@ -46,4 +47,5 @@ namespace MWWorld
             MWBase::Environment::get().getMechanicsManager()->itemTaken(actor, *it, target, itemCount);
-            actorStore.add(*it, itemCount);
+            MWWorld::Ptr received = *actorStore.add(*it, itemCount);
             store.remove(*it, itemCount);
+            MWBase::Environment::get().getLuaManager()->onLorkhanItemPickup(actor, received, itemCount, target);
             std::string name{ it->getClass().getName(*it) };
